scripts/static_server.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
```

Comment:
Yes, it runs a static file server for the browser-based tests. I did look into Node and I can do so again, I think it just didn't offer the out-of-the-box static file server but then I ended up customizing the Python one anyway (originally it was just`python3 -m http.server`).

Reply:
I don't know how many people are going to run the browser tests locally.  Node would be better because it's already necessary elsewhere, but once I enhance the nix-shell, _I_ won't care.

Reply:
Ok, I investigated this again. The [Node.js docs](https://nodejs.org/en/knowledge/HTTP/servers/how-to-serve-static-files/) literally point you to a 3rd-party library for a static file server.

So I think the options here are:
1. Keep using the python3 static server
2. Add an `npm` dependency to `node-static`
3. Try to implement a simple static file server in pure Node.js
4. Implement a Scala/Java static file server as part of the sbt build

Besides sticking with (1), (4) seems the most feasible. Thoughts?

Reply:
1 is fine for now.  Most people either aren't going to run the non-JVM tests, or aren't going to be as stubborn as me when asked to globally install a new interpreter for a language as popular as Python.  We can look at 4 as a separate PR, or I can stop whining and improve the Nix shell.

Reply:
If/when (4) happens, the scaffold server could be rolled into it too. The advantage of having these in the sbt build is they can be more easily started/stopped, currently these scripts have to be started manually before running tests, hope that's okay 😬